HHS/Head-Start-TTADP
# [PROD] TTAHUB-3140 reset TRs to allow goal editing

## Description of change

This is a migration-only PR

TTAHUB-3140: Two TRs have incorrect goal text but sessions have been completed and goals created. This PR resets the Event Reports and Session Reports back to In progress and deletes all the erroneous goals.

## How to test

TTAHUB-3140: The goals on both event reports should now be editable. This is a migration-only PR and the relevant data on Staging is not yet updated to the state that's being corrected (which only changed two days ago). What was done to test was to run the same logic against a different TR on dev [R08-TR-23-8008](https://tta-smarthub-dev.app.cloud.gov/training-report/8008/event-summary) to confirm that it does reset the Event Report and its Session Reports to `In progress`, the TR goal is editable, and the erroneous goals are removed.

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-3140

## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
